README.md

```diff
@@ -74,3 +74,15 @@ In the event that one of the platform applications makes a structural change to
 To update the files compare the embedded resource file with to the newly updated sources ensuring the structure is the same. For `ServiceControl.exe.config` and `ServiceControl.Monitoring.exe.config` ensure the `TransportType` remains configured for the Learning Transport.
 
 When finished, commit the changes to a branch and raise a pull request against master. 
+
+## Deploying
+
+As we don't care about patching older releases, the Platform Sample uses a simplified version of Release Flow that, in most cases, does not require `release-X.Y` branches.
+
+* Builds on the master branch will, by default, create an alpha version of the next minor.
+* To create a production release, label the master branch with the full version number, and trigger a build.
+* Most releases are to support updated versions of platform tools, and these should increment the minor.
+  * Patch releases can be used if a release was broken in some ways. In this case, consider unlisting the offending release from NuGet.
```

Comment:
Where are you saying this would happen? If you're patching from `master`, this is problematic from the standpoint of using the prerelease builds that normally come from master. I believe that the patch tag will reset the git height used at as the prerelease number, so you can end up with duplicate package versions.

You really shouldn't patch from master. You either need to stick to minor/majors, or create the release branch to patch from.